doc/classes/DynamicFont.xml

```diff
@@ -5,6 +5,12 @@
 	</brief_description>
 	<description>
 		DynamicFont renders vector font files (such as TTF or OTF) dynamically at runtime instead of using a prerendered texture atlas like [BitmapFont]. This trades the faster loading time of [BitmapFont]s for the ability to change font parameters like size and spacing during runtime. [DynamicFontData] is used for referencing the font file paths.
+		[codeblock]
+		var dynamic_font = DynamicFont.new()
+		dynamic_font.font_data = load("res://BarlowCondensed-Bold.ttf")
```

Comment:
`load("res://path/to/font.ttf")` would be better for general.

Reply:
Shouldn't we also use `preload()` instead of `load()` in the example?

Reply:
I think this example can be used as loading font on demand.
so, `load()` seems fine to me.